pom.xml

```diff
@@ -404,6 +417,53 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>com.github.ozsie</groupId>
+        <artifactId>detekt-maven-plugin</artifactId>
+        <version>${detekt-maven-plugin.version}</version>
+        <executions>
+          <execution>
+            <phase>verify</phase>
+            <goals><goal>check</goal></goals>
+            <configuration>
+              <report>
+                <report>xml:${project.basedir}/target/site/detekt.xml</report>
+                <report>html:${project.basedir}/target/site/detekt.html</report>
+              </report>
+              <config>detekt.yml</config>
+            </configuration>
+          </execution>
+        </executions>
+        <dependencies>
+          <dependency>
+            <groupId>io.gitlab.arturbosch.detekt</groupId>
+            <artifactId>detekt-rules-libraries</artifactId>
+            <version>${detekt-maven-plugin.version}</version>
+          </dependency>
+        </dependencies>
+      </plugin>
+      <plugin>
+        <groupId>com.github.gantsign.maven</groupId>
+        <artifactId>ktlint-maven-plugin</artifactId>
+        <version>3.0.0</version>
+        <executions>
+          <execution>
+            <id>format-and-check</id>
+            <goals>
+              <goal>check</goal>
+            </goals>
+            <configuration>
+              <failOnViolation>false</failOnViolation>
```

Comment:
I think that the build should fail if the linter's violation occurs. Please, remove it

Reply:
removed